power_grid_model_c/power_grid_model/include/power_grid_model/component/appliance.hpp

```diff
@@ -94,17 +94,17 @@ class Appliance : public Base {
         }
         return output;
     }
-    template <bool sym>
-    ApplianceShortCircuitOutput get_sc_output(ComplexValue<sym> const& i) const {
+    ApplianceShortCircuitOutput get_sc_output(ComplexValue<false> const& i) const {
         ApplianceShortCircuitOutput output{};
         static_cast<BaseOutput&>(output) = base_output(true);
-        // TODO(NITISH) convert sym output
-        if constexpr (!sym) {
-            output.i = base_i_ * cabs(i);
-            output.i_angle = arg(i * injection_direction());
-        }
+        output.i = base_i_ * cabs(i);
+        output.i_angle = arg(i * injection_direction());
         return output;
     }
+    ApplianceShortCircuitOutput get_sc_output(ComplexValue<true> const& i) const {
+        ComplexValue<false> const iabc{i, i * a2, i * a};
```

Comment:
I think we can do it using just one value.

https://github.com/PowerGridModel/power-grid-model/blob/a7eae1a3ad013191bd0a9f1afe074c7f907e7008/power_grid_model_c/power_grid_model/include/power_grid_model/three_phase_tensor.hpp#L49

`ComplexValue<false>` has a constructor which take one complex value, and automatically rotate for three phase. So you don't need to multiply it as you did there.

Please change it for all the places